File: GFG POTD/032-February-2026/009-rotation.java
// https://www.geeksforgeeks.org/problems/rotation4723/1




class Solution {
    public int findKRotation(int arr[]) {
        int n = arr.length;
        for (int i = 1; i < n; i ++) {
            if (arr[i - 1] > arr[i]) {
                return i;
            }
        }
        return 0;
    }
}

File: GFG POTD/032-February-2026/009-rotation.py
# https://www.geeksforgeeks.org/problems/rotation4723/1/



class Solution:
    def findKRotation(self, arr):
        n = len(arr)
        for i in range(1, n):
            if arr[i - 1] > arr[i]:
                return i
        
        return 0